client/debian/control

```diff
@@ -4,24 +4,53 @@ Priority: optional
-Vcs-Git: https://github.com/canonical/hardware-api.git [client/hwlib/src]
-Vcs-Browser: https://github.com/canonical/hardware-api/tree/main/client/hwlib/src
+Vcs-Git: https://github.com/canonical/hardware-api.git [client]
+Vcs-Browser: https://github.com/canonical/hardware-api/tree/main/client
 Homepage: https://github.com/canonical/hardware-api
 X-Cargo-Crate: hwlib
 Rules-Requires-Root: no
 
 Package: librust-hwlib-dev
-Architecture: any
-Multi-Arch: same
-Depends: libssl-dev, pkgconf, ${misc:Depends}, ${shlibs:Depends}
+Architecture: amd64
+Multi-Arch: foreign
+Depends: libssl-dev, pkgconf, python3, ${misc:Depends}, ${shlibs:Depends}
 Provides: librust-hwlib-dev (= ${binary:Version})
-Description: Collects and sends hardware information to hardware-api to find testing status of the machine
- This library is used for gathering hardware information from a device and sending it
- to the hardware-api to determine the testing status of the device model.
+Description: Library for querying Ubuntu hardware certification status - Rust source code
+ A Rust library that collects hardware information from the local
+ system and queries the Ubuntu Hardware Certification service to
+ determine if the device model has been certified.
+ .
+ The library identifies system components, creates a hardware profile, and
+ communicates with the certification database to check:
+  * If the exact or similar device model has been certified
+  * Which Ubuntu releases the device is certified for
+  * Current certification status
+ .
+ This package provides the development files needed to use hwlib in Rust
+ applications.
+
+Package: hwctl
+Architecture: amd64
+Multi-Arch: foreign
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Description: Command-line tool for checking Ubuntu hardware certification status - Rust source code
```

Comment:
Please check if this field should be capitalize. I don't recall now.

Reply:
Seems that all Rust libraries have it capitalised
![image](https://github.com/user-attachments/assets/9ffc41cf-42b4-4eda-8024-8a4c660756e7)

Reply:
Thanks for checking.